chromium/pages/popup/style.css

```diff
@@ -1,128 +1,316 @@
-html {
-  background-color: #fff
+:root {
+  --https-blue: #0a84ff;
+  --darker-blue: #0060df;
+  --text-main: #000;
+  --text-secondary: #464646;
+  --ligh-grey: #ececec;
+  --space: 5px;
 }
 
 body {
-  cursor: default;
-  margin-left: 1em;
-  margin-right: 1em;
-  margin-top: 0;
-  margin-bottom: 0;
-  min-width: 23em;
+  color: var(--text-secondary);
   font-family: 'Lucida Grande', 'Segoe UI', Tahoma, 'DejaVu Sans', Arial, sans-serif;
-  font-size: 75%;
-  color: #303942;
+  margin: var(--margin);
+  max-width: 400px;
+  min-width: 380px;
 }
 
-p {
-  line-height: 1.8em;
+header {
+  background-color: var(--ligh-grey);
+  color: var(--text-main);
+  display: block;
+  padding: var(--space);
 }
 
-h1, h2, h3 {
-  user-select: none;
-  font-weight: normal;
-  line-height: 1;
+header h1 {
+  font-size: 2em;
+}
+
+header .logo-container {
+  width: 100%;
+  float: left;
+}
+
+header .logo-container img {
+  width: 70%;
+}
+
+header small {
+  color: #000;
+  display: block;
+  font-size: .70em;
+  font-weight: bold;
+  margin: 1%;
+  width: 100%;
+}
+
+h1,
+h2,
+h3 {
+  font-size: 1em;
 }
 
 h1 {
-  font-size: 1.5em;
+  margin: var(--space) auto
 }
 
 h2 {
-  font-size: 1.3em;
-  margin-bottom: 0.4em;
+  clear: both;
+  font-weight: normal;
+  margin: 0;
 }
 
-h3 {
-  color: black;
-  font-size: 1.2em;
-  margin-bottom: 0.8em;
+h1,
+h2 {
+  color: var(--text-main);
 }
 
-a {
-  color: #15c;
-  text-decoration: underline;
+footer {
+  margin: var(--space);
 }
 
-a:active {
-  color: #052577;
+/* Elements */
+.settings_block {
+  display: inline-block;
+  padding: 20px 1em;
+  border-bottom: #000 solid 1px;
 }
 
-/* Don't wrap text for important stuff. */
-h1, h2, h3, .rule > label {
-  white-space: nowrap;
+.button {
+  border: var(--https-blue) solid 1px;
+  color: var(--text-main);
+  display: block;
+  font-size: 14px;
+  font-weight: bold;
+  margin: var(--space) auto;
+  padding: 20px;
+  text-align: center;
+  text-decoration: none;
 }
 
-/* Hide rules & options if the extension is off. */
-.disabled #RuleManagement,.disabled #HttpNowhere {
-  display: none;
+.button:hover {
+  background-color: var(--darker-blue);
+  color: #fff
 }
 
-/* Initially hide section (until rules get added). */
-section.rules {
-  display: none;
+.see_more__prompt {
+  color: var(--text-main);
+  cursor: pointer;
+  font-size: .9em;
+  float: right;
 }
 
-.rule {
-  height: 25px;
+.see_more__arrow {
+  border: solid var(--darker-blue);
+  border-width: 0 3px 3px 0;
+  cursor: pointer;
+  float: right;
+  margin-left: 10px;
+  margin-top: var(--space);
+  padding: 3px;
 }
 
-.rule label {
-  padding: 0;
-  max-width: 93%;
-  overflow: hidden;
+.see_more__content.show p {
+  margin: 10px 0 0 0;
 }
 
-/* Underline rules that have notes. */
-.rule[title] span {
-  border-bottom: 1px dotted;
-  cursor: help;
+.down {
+  transform: rotate(45deg);
+  -webkit-transform: rotate(45deg);
```

Comment:
Is the `-webkit-` prefix really necessary? https://caniuse.com/#feat=transforms2d

Reply:
Probably not, old habits from previous web development days. I'll amend this.